dotnet/InkRecognition/wpf-app/src/MainWindow.xaml.cs

```diff
@@ -19,6 +19,7 @@ public partial class MainWindow : Window
         InkRecognizer inkRecognizer;
         DisplayInformation displayInfo;
         private readonly DispatcherTimer dispatcherTimer;
+        // Time to wait before triggering ink recognition operation
```

Comment:
Can you also add a comment before L29 : 
// Replace the subscriptionKey string value with your valid subscription key.